conanfile.py

```diff
@@ -10,20 +9,22 @@ class NaiveTsearchConan(ConanFile):
     homepage = "https://github.com/kulp/naive-tsearch"
     url = "https://github.com/kulp/naive-tsearch"
     license = "MIT"
-    exports_sources = "CMakeLists.txt", "README.md", "*.c", "*.h", "*.in", "*.inc", "LICENSE", "tests/**"
+    exports_sources = "CMakeLists.txt", "create_release.py", "Changelog", "distfiles.yml", "README.md", "NAIVETSEARCH_VERSION", \
+                      "*.c", "*.h", "*.in", "*.inc", "LICENSE", "tests/**"
     exports = "LICENSE"
     no_copy_source = True
     settings = "os", "arch", "compiler", "build_type"
-    requires = "boost/1.73.0"
+    requires = "boost/[>1.70.0]"
```

Comment:
I'm not sure anymore version ranges are well supported by the conan servers.
See https://ci.appveyor.com/project/madebr/naive-tsearch/builds/34064592#L118
It looks like the search for all boost versions, causes high server stress.

I propose to revert this version range change.

Reply:
I'll revert for now and ask around what to do.

Reply:
It is not a big deal to me if a specific version is needed ; more to the point, I was wondering if we could require something less than the latest version (1.73.0). If requesting "boost/[>1.70.0]" is not possible and if requiring "boost/1.70.0" will not allow newer versions than 1.70.0, then requiring "boost/1.73.0" seems fine to me.

Reply:
Conan and CCI is relatively new. By choosing a higher version, I was more certain that its binary packages would get supported/available longer.
I think it is possible to override the boost version to any version you want, even lower.
But don't quote me on that (I haven't checked it).